requirements.txt

```
@@ -12,6 +12,7 @@ numpy>=1.15.0
 requests>=2.13.0,<3.0.0
 plac>=0.9.6,<1.2.0
 pathlib==1.0.1; python_version < "3.4"
+tqdm>=4.38.0
```

Maybe add an upper range pin, just to be safe?

```suggestion
tqdm>=4.38.0,<5.0.0
```

---

Done!